_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,276 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
```

Comment:
Sorry, i missed this before, but there are a couple of ambiguous DOM traversals here:
- with a visible text node <-- has a descendant in the flat tree
- does not have children <-- in the flat tree
- has an ancestor <-- in the flat tree